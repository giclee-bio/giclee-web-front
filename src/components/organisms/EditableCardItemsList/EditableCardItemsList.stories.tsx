import { Meta, StoryObj } from '@storybook/react'

import EditableCardItemsList from './EditableCardItemsList'

const meta: Meta<typeof EditableCardItemsList> = {}

export default meta
type Story = StoryObj<typeof EditableCardItemsList>

export const Index: Story = {
  args: {},
}
