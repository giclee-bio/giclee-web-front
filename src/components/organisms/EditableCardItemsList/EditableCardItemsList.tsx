const EditableCardItemsList = () => {
  return <>EditableCardItemsList</>
}

export default EditableCardItemsList
