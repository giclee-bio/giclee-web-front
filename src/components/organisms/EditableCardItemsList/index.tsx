import EditableCardItemsList from './EditableCardItemsList'

export default EditableCardItemsList
