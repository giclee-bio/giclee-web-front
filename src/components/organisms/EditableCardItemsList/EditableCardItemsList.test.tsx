import { render } from '@testing-library/react'

import EditableCardItemsList from './EditableCardItemsList'

import type { PlatformItems } from '@/components/atoms/PlatformItem'

const item = {
  alt: 'nacal',
  caption: 'nacal',
  src: '/nacal.png',
}
const items: PlatformItems[] = new Array(8).fill(item)

describe('EditableCardItemsList', () => {
  it('snapshot', async () => {
    const { asFragment } = render(
      <EditableCardItemsList handleChangeText={() => jest.mock} items={items} />,
      {}
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
