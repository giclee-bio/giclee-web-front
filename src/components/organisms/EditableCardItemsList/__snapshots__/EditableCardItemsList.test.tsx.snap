// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditableCardItemsList snapshot 1`] = `
<DocumentFragment>
  <form
    class="mt-4 max-w-md"
  >
    <ol
      class="min-h-[320px]"
    >
      <li
        class="mt-6 flex list-none items-center justify-between rounded-lg bg-stone-800 p-4 pl-8 first:mt-0"
      >
        <div
          class=""
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r0:"
              >
                nacal
              </label>
            </div>
            <div
              class="mt-2 flex gap-2"
            >
              <img
                alt="nacal"
                class="rounded"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/_next/image?url=%2Fnacal.png&w=64&q=75"
                srcset="/_next/image?url=%2Fnacal.png&w=32&q=75 1x, /_next/image?url=%2Fnacal.png&w=64&q=75 2x"
                style="color: transparent;"
                width="32"
              />
              <input
                class="bg-stone-50 h-8 w-72 rounded px-3 text-black"
                id="radix-:r0:"
                name="nacal"
                required=""
                title=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="mt-2 flex justify-end"
          >
            <button
              class="text-sm text-ruby-400"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
        <svg
          class="text-stone-400"
          fill="none"
          height="24"
          viewBox="0 0 15 15"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2.49998 4.09998C2.27906 4.09998 2.09998 4.27906 2.09998 4.49998C2.09998 4.72089 2.27906 4.89998 2.49998 4.89998H12.5C12.7209 4.89998 12.9 4.72089 12.9 4.49998C12.9 4.27906 12.7209 4.09998 12.5 4.09998H2.49998ZM2.49998 6.09998C2.27906 6.09998 2.09998 6.27906 2.09998 6.49998C2.09998 6.72089 2.27906 6.89998 2.49998 6.89998H12.5C12.7209 6.89998 12.9 6.72089 12.9 6.49998C12.9 6.27906 12.7209 6.09998 12.5 6.09998H2.49998ZM2.09998 8.49998C2.09998 8.27906 2.27906 8.09998 2.49998 8.09998H12.5C12.7209 8.09998 12.9 8.27906 12.9 8.49998C12.9 8.72089 12.7209 8.89998 12.5 8.89998H2.49998C2.27906 8.89998 2.09998 8.72089 2.09998 8.49998ZM2.49998 10.1C2.27906 10.1 2.09998 10.2791 2.09998 10.5C2.09998 10.7209 2.27906 10.9 2.49998 10.9H12.5C12.7209 10.9 12.9 10.7209 12.9 10.5C12.9 10.2791 12.7209 10.1 12.5 10.1H2.49998Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </li>
      <li
        class="mt-6 flex list-none items-center justify-between rounded-lg bg-stone-800 p-4 pl-8 first:mt-0"
      >
        <div
          class=""
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r1:"
              >
                nacal
              </label>
            </div>
            <div
              class="mt-2 flex gap-2"
            >
              <img
                alt="nacal"
                class="rounded"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/_next/image?url=%2Fnacal.png&w=64&q=75"
                srcset="/_next/image?url=%2Fnacal.png&w=32&q=75 1x, /_next/image?url=%2Fnacal.png&w=64&q=75 2x"
                style="color: transparent;"
                width="32"
              />
              <input
                class="bg-stone-50 h-8 w-72 rounded px-3 text-black"
                id="radix-:r1:"
                name="nacal"
                required=""
                title=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="mt-2 flex justify-end"
          >
            <button
              class="text-sm text-ruby-400"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
        <svg
          class="text-stone-400"
          fill="none"
          height="24"
          viewBox="0 0 15 15"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2.49998 4.09998C2.27906 4.09998 2.09998 4.27906 2.09998 4.49998C2.09998 4.72089 2.27906 4.89998 2.49998 4.89998H12.5C12.7209 4.89998 12.9 4.72089 12.9 4.49998C12.9 4.27906 12.7209 4.09998 12.5 4.09998H2.49998ZM2.49998 6.09998C2.27906 6.09998 2.09998 6.27906 2.09998 6.49998C2.09998 6.72089 2.27906 6.89998 2.49998 6.89998H12.5C12.7209 6.89998 12.9 6.72089 12.9 6.49998C12.9 6.27906 12.7209 6.09998 12.5 6.09998H2.49998ZM2.09998 8.49998C2.09998 8.27906 2.27906 8.09998 2.49998 8.09998H12.5C12.7209 8.09998 12.9 8.27906 12.9 8.49998C12.9 8.72089 12.7209 8.89998 12.5 8.89998H2.49998C2.27906 8.89998 2.09998 8.72089 2.09998 8.49998ZM2.49998 10.1C2.27906 10.1 2.09998 10.2791 2.09998 10.5C2.09998 10.7209 2.27906 10.9 2.49998 10.9H12.5C12.7209 10.9 12.9 10.7209 12.9 10.5C12.9 10.2791 12.7209 10.1 12.5 10.1H2.49998Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </li>
      <li
        class="mt-6 flex list-none items-center justify-between rounded-lg bg-stone-800 p-4 pl-8 first:mt-0"
      >
        <div
          class=""
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r2:"
              >
                nacal
              </label>
            </div>
            <div
              class="mt-2 flex gap-2"
            >
              <img
                alt="nacal"
                class="rounded"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/_next/image?url=%2Fnacal.png&w=64&q=75"
                srcset="/_next/image?url=%2Fnacal.png&w=32&q=75 1x, /_next/image?url=%2Fnacal.png&w=64&q=75 2x"
                style="color: transparent;"
                width="32"
              />
              <input
                class="bg-stone-50 h-8 w-72 rounded px-3 text-black"
                id="radix-:r2:"
                name="nacal"
                required=""
                title=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="mt-2 flex justify-end"
          >
            <button
              class="text-sm text-ruby-400"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
        <svg
          class="text-stone-400"
          fill="none"
          height="24"
          viewBox="0 0 15 15"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2.49998 4.09998C2.27906 4.09998 2.09998 4.27906 2.09998 4.49998C2.09998 4.72089 2.27906 4.89998 2.49998 4.89998H12.5C12.7209 4.89998 12.9 4.72089 12.9 4.49998C12.9 4.27906 12.7209 4.09998 12.5 4.09998H2.49998ZM2.49998 6.09998C2.27906 6.09998 2.09998 6.27906 2.09998 6.49998C2.09998 6.72089 2.27906 6.89998 2.49998 6.89998H12.5C12.7209 6.89998 12.9 6.72089 12.9 6.49998C12.9 6.27906 12.7209 6.09998 12.5 6.09998H2.49998ZM2.09998 8.49998C2.09998 8.27906 2.27906 8.09998 2.49998 8.09998H12.5C12.7209 8.09998 12.9 8.27906 12.9 8.49998C12.9 8.72089 12.7209 8.89998 12.5 8.89998H2.49998C2.27906 8.89998 2.09998 8.72089 2.09998 8.49998ZM2.49998 10.1C2.27906 10.1 2.09998 10.2791 2.09998 10.5C2.09998 10.7209 2.27906 10.9 2.49998 10.9H12.5C12.7209 10.9 12.9 10.7209 12.9 10.5C12.9 10.2791 12.7209 10.1 12.5 10.1H2.49998Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </li>
      <li
        class="mt-6 flex list-none items-center justify-between rounded-lg bg-stone-800 p-4 pl-8 first:mt-0"
      >
        <div
          class=""
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r3:"
              >
                nacal
              </label>
            </div>
            <div
              class="mt-2 flex gap-2"
            >
              <img
                alt="nacal"
                class="rounded"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/_next/image?url=%2Fnacal.png&w=64&q=75"
                srcset="/_next/image?url=%2Fnacal.png&w=32&q=75 1x, /_next/image?url=%2Fnacal.png&w=64&q=75 2x"
                style="color: transparent;"
                width="32"
              />
              <input
                class="bg-stone-50 h-8 w-72 rounded px-3 text-black"
                id="radix-:r3:"
                name="nacal"
                required=""
                title=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="mt-2 flex justify-end"
          >
            <button
              class="text-sm text-ruby-400"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
        <svg
          class="text-stone-400"
          fill="none"
          height="24"
          viewBox="0 0 15 15"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2.49998 4.09998C2.27906 4.09998 2.09998 4.27906 2.09998 4.49998C2.09998 4.72089 2.27906 4.89998 2.49998 4.89998H12.5C12.7209 4.89998 12.9 4.72089 12.9 4.49998C12.9 4.27906 12.7209 4.09998 12.5 4.09998H2.49998ZM2.49998 6.09998C2.27906 6.09998 2.09998 6.27906 2.09998 6.49998C2.09998 6.72089 2.27906 6.89998 2.49998 6.89998H12.5C12.7209 6.89998 12.9 6.72089 12.9 6.49998C12.9 6.27906 12.7209 6.09998 12.5 6.09998H2.49998ZM2.09998 8.49998C2.09998 8.27906 2.27906 8.09998 2.49998 8.09998H12.5C12.7209 8.09998 12.9 8.27906 12.9 8.49998C12.9 8.72089 12.7209 8.89998 12.5 8.89998H2.49998C2.27906 8.89998 2.09998 8.72089 2.09998 8.49998ZM2.49998 10.1C2.27906 10.1 2.09998 10.2791 2.09998 10.5C2.09998 10.7209 2.27906 10.9 2.49998 10.9H12.5C12.7209 10.9 12.9 10.7209 12.9 10.5C12.9 10.2791 12.7209 10.1 12.5 10.1H2.49998Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </li>
      <li
        class="mt-6 flex list-none items-center justify-between rounded-lg bg-stone-800 p-4 pl-8 first:mt-0"
      >
        <div
          class=""
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r4:"
              >
                nacal
              </label>
            </div>
            <div
              class="mt-2 flex gap-2"
            >
              <img
                alt="nacal"
                class="rounded"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/_next/image?url=%2Fnacal.png&w=64&q=75"
                srcset="/_next/image?url=%2Fnacal.png&w=32&q=75 1x, /_next/image?url=%2Fnacal.png&w=64&q=75 2x"
                style="color: transparent;"
                width="32"
              />
              <input
                class="bg-stone-50 h-8 w-72 rounded px-3 text-black"
                id="radix-:r4:"
                name="nacal"
                required=""
                title=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="mt-2 flex justify-end"
          >
            <button
              class="text-sm text-ruby-400"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
        <svg
          class="text-stone-400"
          fill="none"
          height="24"
          viewBox="0 0 15 15"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2.49998 4.09998C2.27906 4.09998 2.09998 4.27906 2.09998 4.49998C2.09998 4.72089 2.27906 4.89998 2.49998 4.89998H12.5C12.7209 4.89998 12.9 4.72089 12.9 4.49998C12.9 4.27906 12.7209 4.09998 12.5 4.09998H2.49998ZM2.49998 6.09998C2.27906 6.09998 2.09998 6.27906 2.09998 6.49998C2.09998 6.72089 2.27906 6.89998 2.49998 6.89998H12.5C12.7209 6.89998 12.9 6.72089 12.9 6.49998C12.9 6.27906 12.7209 6.09998 12.5 6.09998H2.49998ZM2.09998 8.49998C2.09998 8.27906 2.27906 8.09998 2.49998 8.09998H12.5C12.7209 8.09998 12.9 8.27906 12.9 8.49998C12.9 8.72089 12.7209 8.89998 12.5 8.89998H2.49998C2.27906 8.89998 2.09998 8.72089 2.09998 8.49998ZM2.49998 10.1C2.27906 10.1 2.09998 10.2791 2.09998 10.5C2.09998 10.7209 2.27906 10.9 2.49998 10.9H12.5C12.7209 10.9 12.9 10.7209 12.9 10.5C12.9 10.2791 12.7209 10.1 12.5 10.1H2.49998Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </li>
      <li
        class="mt-6 flex list-none items-center justify-between rounded-lg bg-stone-800 p-4 pl-8 first:mt-0"
      >
        <div
          class=""
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r5:"
              >
                nacal
              </label>
            </div>
            <div
              class="mt-2 flex gap-2"
            >
              <img
                alt="nacal"
                class="rounded"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/_next/image?url=%2Fnacal.png&w=64&q=75"
                srcset="/_next/image?url=%2Fnacal.png&w=32&q=75 1x, /_next/image?url=%2Fnacal.png&w=64&q=75 2x"
                style="color: transparent;"
                width="32"
              />
              <input
                class="bg-stone-50 h-8 w-72 rounded px-3 text-black"
                id="radix-:r5:"
                name="nacal"
                required=""
                title=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="mt-2 flex justify-end"
          >
            <button
              class="text-sm text-ruby-400"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
        <svg
          class="text-stone-400"
          fill="none"
          height="24"
          viewBox="0 0 15 15"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2.49998 4.09998C2.27906 4.09998 2.09998 4.27906 2.09998 4.49998C2.09998 4.72089 2.27906 4.89998 2.49998 4.89998H12.5C12.7209 4.89998 12.9 4.72089 12.9 4.49998C12.9 4.27906 12.7209 4.09998 12.5 4.09998H2.49998ZM2.49998 6.09998C2.27906 6.09998 2.09998 6.27906 2.09998 6.49998C2.09998 6.72089 2.27906 6.89998 2.49998 6.89998H12.5C12.7209 6.89998 12.9 6.72089 12.9 6.49998C12.9 6.27906 12.7209 6.09998 12.5 6.09998H2.49998ZM2.09998 8.49998C2.09998 8.27906 2.27906 8.09998 2.49998 8.09998H12.5C12.7209 8.09998 12.9 8.27906 12.9 8.49998C12.9 8.72089 12.7209 8.89998 12.5 8.89998H2.49998C2.27906 8.89998 2.09998 8.72089 2.09998 8.49998ZM2.49998 10.1C2.27906 10.1 2.09998 10.2791 2.09998 10.5C2.09998 10.7209 2.27906 10.9 2.49998 10.9H12.5C12.7209 10.9 12.9 10.7209 12.9 10.5C12.9 10.2791 12.7209 10.1 12.5 10.1H2.49998Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </li>
      <li
        class="mt-6 flex list-none items-center justify-between rounded-lg bg-stone-800 p-4 pl-8 first:mt-0"
      >
        <div
          class=""
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r6:"
              >
                nacal
              </label>
            </div>
            <div
              class="mt-2 flex gap-2"
            >
              <img
                alt="nacal"
                class="rounded"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/_next/image?url=%2Fnacal.png&w=64&q=75"
                srcset="/_next/image?url=%2Fnacal.png&w=32&q=75 1x, /_next/image?url=%2Fnacal.png&w=64&q=75 2x"
                style="color: transparent;"
                width="32"
              />
              <input
                class="bg-stone-50 h-8 w-72 rounded px-3 text-black"
                id="radix-:r6:"
                name="nacal"
                required=""
                title=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="mt-2 flex justify-end"
          >
            <button
              class="text-sm text-ruby-400"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
        <svg
          class="text-stone-400"
          fill="none"
          height="24"
          viewBox="0 0 15 15"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2.49998 4.09998C2.27906 4.09998 2.09998 4.27906 2.09998 4.49998C2.09998 4.72089 2.27906 4.89998 2.49998 4.89998H12.5C12.7209 4.89998 12.9 4.72089 12.9 4.49998C12.9 4.27906 12.7209 4.09998 12.5 4.09998H2.49998ZM2.49998 6.09998C2.27906 6.09998 2.09998 6.27906 2.09998 6.49998C2.09998 6.72089 2.27906 6.89998 2.49998 6.89998H12.5C12.7209 6.89998 12.9 6.72089 12.9 6.49998C12.9 6.27906 12.7209 6.09998 12.5 6.09998H2.49998ZM2.09998 8.49998C2.09998 8.27906 2.27906 8.09998 2.49998 8.09998H12.5C12.7209 8.09998 12.9 8.27906 12.9 8.49998C12.9 8.72089 12.7209 8.89998 12.5 8.89998H2.49998C2.27906 8.89998 2.09998 8.72089 2.09998 8.49998ZM2.49998 10.1C2.27906 10.1 2.09998 10.2791 2.09998 10.5C2.09998 10.7209 2.27906 10.9 2.49998 10.9H12.5C12.7209 10.9 12.9 10.7209 12.9 10.5C12.9 10.2791 12.7209 10.1 12.5 10.1H2.49998Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </li>
      <li
        class="mt-6 flex list-none items-center justify-between rounded-lg bg-stone-800 p-4 pl-8 first:mt-0"
      >
        <div
          class=""
        >
          <div
            class="mt-4 first:mt-0"
          >
            <div
              class="flex justify-between text-sm"
            >
              <label
                class="font-bold"
                for="radix-:r7:"
              >
                nacal
              </label>
            </div>
            <div
              class="mt-2 flex gap-2"
            >
              <img
                alt="nacal"
                class="rounded"
                data-nimg="1"
                decoding="async"
                height="32"
                loading="lazy"
                src="/_next/image?url=%2Fnacal.png&w=64&q=75"
                srcset="/_next/image?url=%2Fnacal.png&w=32&q=75 1x, /_next/image?url=%2Fnacal.png&w=64&q=75 2x"
                style="color: transparent;"
                width="32"
              />
              <input
                class="bg-stone-50 h-8 w-72 rounded px-3 text-black"
                id="radix-:r7:"
                name="nacal"
                required=""
                title=""
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            class="mt-2 flex justify-end"
          >
            <button
              class="text-sm text-ruby-400"
              type="button"
            >
              Delete
            </button>
          </div>
        </div>
        <svg
          class="text-stone-400"
          fill="none"
          height="24"
          viewBox="0 0 15 15"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M2.49998 4.09998C2.27906 4.09998 2.09998 4.27906 2.09998 4.49998C2.09998 4.72089 2.27906 4.89998 2.49998 4.89998H12.5C12.7209 4.89998 12.9 4.72089 12.9 4.49998C12.9 4.27906 12.7209 4.09998 12.5 4.09998H2.49998ZM2.49998 6.09998C2.27906 6.09998 2.09998 6.27906 2.09998 6.49998C2.09998 6.72089 2.27906 6.89998 2.49998 6.89998H12.5C12.7209 6.89998 12.9 6.72089 12.9 6.49998C12.9 6.27906 12.7209 6.09998 12.5 6.09998H2.49998ZM2.09998 8.49998C2.09998 8.27906 2.27906 8.09998 2.49998 8.09998H12.5C12.7209 8.09998 12.9 8.27906 12.9 8.49998C12.9 8.72089 12.7209 8.89998 12.5 8.89998H2.49998C2.27906 8.89998 2.09998 8.72089 2.09998 8.49998ZM2.49998 10.1C2.27906 10.1 2.09998 10.2791 2.09998 10.5C2.09998 10.7209 2.27906 10.9 2.49998 10.9H12.5C12.7209 10.9 12.9 10.7209 12.9 10.5C12.9 10.2791 12.7209 10.1 12.5 10.1H2.49998Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </li>
    </ol>
    <button
      class="mx-auto mt-8 block h-10 rounded text-center font-bold leading-10 w-64 bg-primary text-black"
    >
      保存
    </button>
  </form>
</DocumentFragment>
`;
